react/src/tests/Security-test.js

```diff
@@ -1263,7 +1263,7 @@ describe("Security Container", () => {
     expect(dispatch.mock.calls.length).toEqual(0);
     const wrapper = getWrapper();
     wrapper.find('EduIDButton#security-u2f-button').simulate('click');
-    expect(dispatch.mock.calls.length).toEqual(1);
+    expect(dispatch.mock.calls.length).toEqual(2);
```

Comment:
I would add

```js
expect(dispatch.mock.calls[0][0].type).toEqual("RM_ALL_NOTIFICATION");
expect(dispatch.mock.calls[1][0].type).toEqual(actions.START_ASK_U2F_DESCRIPTION);
```

to make sure these are the two action that I actually wanted to dispatch (and not some other random actions). Any future change would catch that something is different and then we can be sure that the problem won't show up again ;)

Reply:
I would also replace all hardcoded strings with references to the defined consts under the actions, as is already the case with the security-related notification types (`actions/Secury.js`)

Reply:
Done :)

In general these tests might benefit from some work - some cleanup, making them more robust in the sense you point out, and also improving the coverage. It's in the pipeline :)